gemma-web/src/main/webapp/scripts/api/entities/analysis/differentialExpression/DifferentialExpressionAnalysesSummaryTree.js

```diff
@@ -284,7 +284,7 @@ Gemma.DifferentialExpressionAnalysesSummaryTree = Ext
 
             getSubsetText: function (analysis) {
                 var subsetText = '';
-                if (analysis.subsetFactor) {
+                if (analysis.subsetFactor && analysis.subsetFactorValue.factorValue !='DE_Include') {

```

Comment:
These changes shouldn't be in there.

You can use `git rebase -i` to remove those commits.